pom.xml

```diff
@@ -78,7 +78,7 @@ THE SOFTWARE.
     <properties>
         <revision>2.0.8</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.361.4</jenkins.version>
+        <jenkins.version>2.403</jenkins.version>
```

Comment:
I chose 2.403 because that was when the changes were first introduced. Most recent LTS is 2.401.3.

Reply:
`2.414` will become the lts line and there is a bom for this line (which you also should update!)

https://github.com/jenkinsci/ec2-plugin/pull/875/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R222-R223